user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -98,16 +126,20 @@ export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionPr
   );
   const [foundValidCaseNumber, setFoundValidCaseNumber] = useState<boolean>(false);
   const [isConsolidationProcessing, setIsConsolidationProcessing] = useState<boolean>(false);
-  const [isDataEnhanced, setIsDataEnhanced] = useState<boolean>(false);
+  const [areAttorneysFetched, setAreAttorneysFetched] = useState<boolean>(false);
```

Comment:
This was poorly named I felt.  The only thing it's used for is retrieving attorneys.